skbeam/core/fitting/base/__init__.py

```diff
@@ -36,6 +36,6 @@
 # POSSIBILITY OF SUCH DAMAGE.                                          #
 ########################################################################
 from __future__ import absolute_import, division, print_function
-import six
+import six  # noqa: F401
```

Comment:
We can remove six imports at this point.

Reply:
The import is removed